opensrp-chw-core/src/main/java/org/smartregister/chw/core/service/StockUsageReportService.java

```diff
@@ -83,11 +83,11 @@ private String getFormSubmissionID(StockUsage usage) {
     }
 
     private void addEvent(JSONObject form, JSONArray jsonArray, StockUsage usage, AllSharedPreferences allSharedPreferences, String formSubmissionId) throws JSONException {
-        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.StockUsageReport.STOCK_NAME, usage.getStockName());
-        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.StockUsageReport.STOCK_YEAR, usage.getYear());
-        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.StockUsageReport.STOCK_MONTH, usage.getMonth());
-        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.StockUsageReport.STOCK_USAGE, usage.getStockUsage());
-        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.StockUsageReport.STOCK_PROVIDER, usage.getProviderId());
+        FormUtils.updateFormField(jsonArray, CoreConstants.JsonAssets.STOCK_NAME, usage.getStockName());
```

Comment:
Can we have all these optimized into 1 cycle? Populating the field values using only 1 loop will improve performance.